packages/svelte/src/internal/client/dom/elements/misc.js

```diff
@@ -56,3 +57,17 @@ export function add_form_reset_listener() {
 		);
 	}
 }
+
+/**
+ * @param {string} text
+ */
+export function title(text) {
+	render_effect(() => {
```

Comment:
Why make this a render effect? Nothing inside it can rerun, because you pass static text. If there's a reason then it could use a comment

Reply:
Ah, this was discussed already, I forgot - continuing the conversation there, because I think this does nothing